src/problem1.rb

```diff
@@ -0,0 +1,12 @@
+
+# Given an integer array of size N,
+# compute the sum of all even numbers in this array
+class Problem1
+  # @param [Array] numbers
+  def self.sum_of_even_numbers(numbers)
+    return 0 if numbers.nil? || numbers.empty?
```

Comment:
Assume you won't be getting any `nil` input as per problem description. Do you need the `empty?` check?

Reply:
I assumed that the function will be used by unknown provider... so that's why I wrote those checks.